src/strategies/squid-dao/schema.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
@ChaituVR Did you tested this schema? We should test it against examples.json

Reply:
Yes, the tests pass!

```
npm run test --strategy=squid-dao                                                            09:38:44

> @snapshot-labs/strategies@0.1.0 pretest
> npm run build


> @snapshot-labs/strategies@0.1.0 build
> tsc -p .


> @snapshot-labs/strategies@0.1.0 postbuild
> copyfiles -u 1 src/**/*.md dist/ && copyfiles -u 1 src/**/*.json dist/


> @snapshot-labs/strategies@0.1.0 test
> jest test/index.spec.ts --runInBand --testTimeout=120000

  console.log
    [
      {
        '0xfd4f24676ed4588928213f37b126b53c07186f45': 5.1986,
        '0x0f763341b448bb0f02370f4037fe4a2c84c9283f': 94.8013
      }
    ]

      at Object.<anonymous> (test/index.spec.ts:47:13)

  console.log
    Resolved in 1.22 sec.

      at Object.<anonymous> (test/index.spec.ts:48:13)

  console.log
    Scores with latest snapshot [
      {
        '0xfd4f24676ed4588928213f37b126b53c07186f45': 5.1925,
        '0x0f763341b448bb0f02370f4037fe4a2c84c9283f': 94.8074
      }
    ]

      at Object.<anonymous> (test/index.spec.ts:110:13)

  console.log
    Resolved in 0.84 sec.

      at Object.<anonymous> (test/index.spec.ts:111:13)

 PASS  test/index.spec.ts (5.197 s)
  
Test strategy "squid-dao"
    ✓ Strategy should run without any errors (1236 ms)
    ✓ Should return an array of object with addresses (2 ms)
    ✓ Should take less than 10 sec. to resolve (1 ms)
    ✓ File examples.json should include at least 1 address with a positive score
    ✓ File examples.json must use a snapshot block number in the past (569 ms)
    ✓ File examples.json must have symbol in its strategy params
    ✓ Returned addresses should be either same case as input addresses or checksum addresses (1 ms)
  
Test strategy "squid-dao" with latest snapshot
    ✓ Strategy should run without any errors (1338 ms)
    ✓ Should return an array of object with addresses (1 ms)
  
Test strategy "squid-dao" (with 500 addresses)
    ○ skipped Should work with 500 addresses
    ○ skipped Should take less than 20 sec. to resolve with 500 addresses
  
Others:
    ✓ Author in strategy should be a valid github username (200 ms)
    ✓ Version in strategy should be a valid string

Test Suites: 1 passed, 1 total
Tests:       2 skipped, 11 passed, 13 total
Snapshots:   0 total
Time:        6.054 s
```

Reply:
@bonustrack Yes validated the schema against examples.json and no errors in it :) should be good to go